packages/react-components/react-migration-v0-v9/package.json

```diff
@@ -27,14 +27,14 @@
     "@fluentui/eslint-plugin": "*",
     "@fluentui/react-conformance": "*",
     "@fluentui/react-conformance-griffel": "*",
-    "@fluentui/react-northstar": "*",
     "@fluentui/scripts-api-extractor": "*",
     "@fluentui/scripts-tasks": "*",
     "@fluentui/scripts-storybook": "*"
   },
   "dependencies": {
     "@fluentui/react-aria": "^9.7.0",
     "@fluentui/react-components": "^9.44.3",
+    "@fluentui/react-northstar": "^0.66.4",
```

Comment:
we've had problems with taking on northstar as a direct dependency in Teams, let's add it as a peer dep, since any customers using this package will already have northstar installed

Reply:
gotcha, what version range should be used for peer dep ?